beacon-chain/blockchain/fork_choice.go

```diff
@@ -148,14 +149,28 @@ func (c *ChainService) ApplyForkChoiceRule(
 	if err != nil {
 		return fmt.Errorf("could not hash head block: %v", err)
 	}
+
+	chainHead, err := c.beaconDB.ChainHead()
+	if err != nil {
+		return fmt.Errorf("could not retrieve chain head: %v", err)
+	}
+	chainRoot, err := hashutil.HashBeaconBlock(chainHead)
```

Comment:
`headRoot` sounds better here

Reply:
we already have a `headRoot` above, thats why I used chainHead to differentiate it